### DISCUSSION TITLE
Scrolling diagonally

### DISCUSSION THREAD

**Discussion by carnoxen** [community/community#147612] [Mobile]
Why i can't scroll code pages diagonally? It is distracting to look the page.

**Comment by xDweeb** [comment]
It would indeed be helpful to enable diagonal scrolling for code pages. Sometimes, the lack of this feature makes it challenging to review or compare longer lines of code while keeping track of vertical alignment. Allowing diagonal scrolling could reduce distractions and improve the readability of code in some situations.
